aurorastation.dme

```diff
@@ -593,7 +593,7 @@
 #include "code\datums\uplink\exosuit_equipment.dm"
 #include "code\datums\uplink\gear loadout.dm"
 #include "code\datums\uplink\grenades.dm"
-#include "code\datums\uplink\hardsuit_modules.dm"
+#include "code\datums\uplink\hardsuits and modules.dm"
```

Comment:
no spaces in path names

I know the rest of this folder breaks that convention -- those should be fixed as well